mnist/serving/base/service.yaml

```diff
@@ -36,4 +36,4 @@ spec:
     targetPort: 8500
   selector:
     app: mnist
-  type: ClusterIP
+  type: LoadBalancer
```

Comment:
This will expose the service on a public IP. We generally don't want to expose services on publicly accessible IP addresses by default because its unsafe. 

By default services should only be exposed through the Kubeflow ISTIO gateway so that it is secured by whatever auth is added to the gateway

Can we define a VirtualService to expose it through the ingress gateway?

Can we tell users to port-forward to the port rather than exposing it behind an LB?

Reply:
Yeah, that makes sense. Will change the service type.